server/app.ts

```diff
@@ -69,7 +69,7 @@ app.post('/api/register', async (req: express.Request, res: express.Response) =>
   }
 });
 
-// Add message endpoint with validation
+// Add message submission endpoint
```

Comment:
In this endpoint optional doesn't allow null. But parentId should be allowed to be null. I'm getting a 400 when I try to test because parentId is required.